dpsim-models/src/DP/DP_Ph3_Resistor.cpp

```diff
@@ -56,147 +56,12 @@ void DP::Ph3::Resistor::mnaCompInitialize(Real omega, Real timeStep,
 
 void DP::Ph3::Resistor::mnaCompApplySystemMatrixStamp(
     SparseMatrixRow &systemMatrix) {
+  MatrixComp conductance = Matrix::Zero(3, 3);
```

Comment:
Maybe think of using the template here
https://github.com/sogno-platform/dpsim/blob/f71d1a3e94d5e0ca2773acc5f4f0c53dac4ce9c8/dpsim-models/include/dpsim-models/Definitions.h#L103-L105